### PR TITLE
Improved Documentation.

### LINKED ISSUE #9: Unlear documentation for `holy-books-quran-translation` variable
**Describe the bug**
It is not at all clear how to get alternate translations of the Quran;
in contrast, the way to get alternate versions of the Bible is fully
documented in `holy-books-bible-version`.

**To Reproduce**

0. The link mentioned in the documentation, https://quran.api-docs.io/v3/options/translations, does not easily show how to get the translation codes.

1. Perhaps @ashfaqfarooqui could tackle this ticket or provide direction.

**Expected behavior**
As with `holy-books-bible-version`, it would be idle if users
could have the list of possible Quranic translations directly
in the `holy-books-quran-translation` variable ---although mention of the Quran.com API should be kept around for interested developers ...it would be nice if  @ashfaqfarooqui could mention **how** we could get such a list of translations, so others could reproduce this later on.

**Additional context**
If this issue becomes stale, the `holy-books-quran-translation` variable may be scrapped; or, at least, not mentioned in the top level user documentation.

### PR DESCRIPTION
**Pull request by ashfaqfarooqui**
Provided a list of code for available english translations.

----

fixes #9 

But I think in the long run we should automate fetching this list.

### CODE REVIEW

File: holy-books.org [hunk 1]
```diff
@@ -525,12 +525,23 @@ a special key ‘:name’ whose value is the Arabic-English name of the chapter.
...
+131   Dr. Mustafa Khattab, the Clear Quran (Default)
+20    Sahih International
+85    Abdul Haleem
+19    Picktall
+22    Yusuf Ali
+95    Abul Ala Maududi
+167   Maarif-ul-Quran
+57    Transliteration
 
-        131 -- Dr.  Mustafa Khattab, the Clear Quran
```
Comment (alhassy) [inline holy-books.org:530]: @ashfaqfarooqui Maybe we want to keep this URL around so that others are aware of it ;-)

File: holy-books.org [hunk 2]
```diff
@@ -625,50 +625,72 @@ E.g. Quran:7:157 results in text “Quran 7:157” with a tooltip showing the ve
...
-                      (format "https://quran.com/%s" chapter)
-                      chapter verse)))
-  :face '(:foreground "green" :weight bold))
+                       (holy-books-quran chapter verse)
+                       (cl-getf (cl-getf holy-books-quran-cache chapter) :name)
+                       (format "https://quran.com/%s" chapter)
+                       chapter verse))
+              ((eq 'md backend)
+               (format "[Quran %s:%s](%s \"﴾%s﴾\n\n %s\")" chapter verse
+                       (format "https://quran.com/%s" chapter)
```
Comment (alhassy) [inline holy-books.org:688]: This and the next 3 lines are repeated above, maybe a `let` clause to factor out the similarities ;-)

File: holy-books.org (hunk 1 above)
Comment (ashfaqfarooqui) [reply holy-books.org:530]: Sure, I'll do that.

Here is a better link, but for some reason, the code for Dr. Mustafa Khattab does not match up; and 167 is missing. 
https://api.quran.com/api/v3/options/translations

File: holy-books.org (hunk 2 above)
Comment (ashfaqfarooqui) [reply holy-books.org:688]: OH! I didn't realize this part got pushed as well. I was planning another PR to add markdown support. 

But, I'll fix this as well  :)

File: holy-books.org (hunk 1 above)
Comment (alhassy) [reply holy-books.org:530]: Woah; we can definitely automate retrieval of such a list with a bit of `s-split` ;-)

File: holy-books.org (hunk 2 above)
Comment (alhassy) [reply holy-books.org:688]: I'm looking forward to it! Keep up the great work :-)

**Pull request merged**